src/data/markdown/translated-guides/en/02 Using k6/12 Execution context variables.md

```diff
@@ -31,17 +31,24 @@ $ k6 run --vus 10 --iterations 100 script.js
 
 ## \_\_VU and \_\_ITER
 
-**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes
-available to the test script.
+**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes available to the test script.
 
 ### \_\_ITER
 
 A numeric counter with the current iteration number for a specific VU. Zero-based.
 
 ### \_\_VU
 
-Current VU number. The value is assigned incrementally for each new VU instance. One-based.
-However, VU number is 0 while executing the setup and teardown functions.
+Current VU number in use. The value is assigned incrementally for each new VU instance, starting from one. The variable will be 0 while executing the setup and teardown functions.
+
+It's important to remember that this value is shown _per server instance running you test_. How many servers we use depends on the size of your test and how you spread it across different load zones. So for a larger test, the value of `__VU` will likely not reflect the actual _total_ number of VUs in your test at a given moment.
```

Comment:
This makes it sound as k6 OSS supports multiple servers running. 
I do think this should be in a *sub* section that is called "Cloud considerations" or something like that so it's pretty obvious it is currently only about the cloud.

Reply:
Good point. I restructured the text to include all the k6-cloud specific execution context envvar info under one header.